recipe/recipe.yaml

```diff
@@ -0,0 +1,577 @@
+context:
+  version: "7.8.3"
+
+recipe:
+  name: suitesparse
+  version: ${{ version }}
+
+source:
+  url: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v${{ version }}.tar.gz
+  sha256: ce39b28d4038a09c14f21e02c664401be73c0cb96a9198418d6a98a7db73a259
+
+build:
+  number: 2
+  string: ss${{ version | replace('.', '') }}_h${{ hash }}
```

Comment:
What's the reason to duplicate the version into the build string?

Reply:
suitesparse is a 'suite' of related but independent libraries. The suite has a version, but each library _also_ has its own version. This just makes it easier to see that `libumfpack 6.3.5` was built as part of `libsuitesparse 7.8.3`. Not all libraries will get any version bump when there is a new suitesparse release, so there may be an ss783 build of libumfpack 6.3.5 _and_ an `ss784` build of libumfpack 6.3.5. I don't expect this will be used for dependency restrictions or anything, it is mostly for clarity and to ensure each build is uploaded even if the library version and hash don't change.